main.go

```diff
@@ -201,6 +206,16 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err = (&controllers.TrivyVulnerabilityReportReconciler{
```

Comment:
We should make this flaggable, I think.
Like the rest of the features, turn it into something we can turn on/off with flags.